src/main/webapp/calendar-script.js

```diff
@@ -43,8 +66,95 @@ function getEvents(calendar) {
         title: event.name,
         start: event.strStartTime,
         end: event.strEndTime,
-        allDay: false
+        allDay: false,
+        extendedProps: {
+          stringDate: event.strStartTime,
+          address: event.address,
+          placeId: event.placeId, 
+          openTime: '9AM',
+          closeTime: '5PM',
+        }
       });
     });
   });
 }
+
+function createMap(modalBody, eventObj) {
+  const infoDisplay = document.createElement('div');
+  let address = document.createElement('p');
+  var urlFormatAddress = eventObj.extendedProps.address.replace(/\s/g, '+')
+  var url = 'https://www.google.com/maps/search/?api=1&query=' + urlFormatAddress + '&query_place_id=' + eventObj.extendedProps.placeId;
+  infoDisplay.innerHTML = '<b>Address: </b><a href=' + url + '>' + eventObj.extendedProps.address + '</a>' + '<br>';
+  infoDisplay.appendChild(address);
+  modalBody.appendChild(infoDisplay)
```

Comment:
I don't think this code is doing what you expect. The `address` `p` element / object is never actually used, but it's added to `infoDisplay` -- all this is doing is adding a space between the address line and the open / close lines.

Also, you are setting `infoDisplay.innerHTML` in order to add the bolded "Address:" and the `a` tag. My preference would be to actually build the elements in JavaScript, and append them as children like you do in the rest of this function. I'll let you make that decision though, since I'm not sure if one is particularly more proper than the other.